### ISSUE TITLE
Rephrase config.yml 'avahi: true' option

### ISSUE THREAD

**Issue by l3ku** [Seravo/wordpress#116]
The actual meaning of the option is currently to expose the Vagrant box on OS X to the local network via bridge interface:
https://github.com/Seravo/wordpress/blob/54b867da64991451487fee23fbef0f41d6ffbd19/Vagrantfile#L89-L99
As we currently already always use Avahi in the box by default, we should maybe rephrase this option to be more logical, like `public: true` or something. We could also consider either dropping this feature or also enable the feature for Linux depending on the actual value this feature currently provides for our customers.

**Comment by ottok** [comment]
The question is, what would be a better name then? If it is changed, it needs to be coordinated so that the Vagrantfile, config-example.yml in the project template and also scripts inside the Vagrant box all update at the same time, and there needs to be enough documentation in place so those who use the current setting name knows to update their config.yml files.
